src/cc/bcc_proc.c

```diff
@@ -121,25 +121,21 @@ static char *_procutils_memfd_path(const int pid, const uint64_t inum) {
   return path;
 }
 
-int bcc_procutils_each_module(int pid, bcc_procutils_modulecb callback,
-                              void *payload) {
-  char procmap_filename[128];
-  FILE *procmap;
-  snprintf(procmap_filename, sizeof(procmap_filename), "/proc/%ld/maps",
-           (long)pid);
-  procmap = fopen(procmap_filename, "r");
-  if (!procmap)
-    return -1;
-
-  char buf[PATH_MAX + 1], perm[5], dev[8];
+// return: 0 -> stopped iterating over modules, no need to look at perf maps
```

Comment:
Is this comment still valid? I don't see the return value checked in `bcc_procutils_each_module`

Reply:
fixed in latest commit